core/types/transaction_marshalling.go

```diff
@@ -200,6 +225,86 @@ func (t *Transaction) UnmarshalJSON(input []byte) error {
 		}
 		etx.Sender = *dec.Sender
 
+	case InternalToExternalTxType:
+		var itx InternalToExternalTx
+		inner = &itx
+		if dec.AccessList == nil {
+			return errors.New("missing required field 'accessList' in internal transaction")
```

Comment:
Should probably update these error messages to say `internal to external' type

Reply:
Updated